src/App.tsx

```diff
@@ -1,14 +1,74 @@
-/* eslint-disable max-len */
-import React from 'react';
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+const getVisibleTodos = (
+  todos: Todo[],
+  filterType: string,
```

Comment:
```suggestion
  filterType: FilterType,
```

Why not to use enum?